pkg/controller/migration/controller.go

```diff
@@ -103,55 +97,42 @@ var _ reconcile.Reconciler = &Reconciler{}
 //
 // Reconciles a Migration object.
 type Reconciler struct {
-	client.Client
-	scheme *runtime.Scheme
-	log    *logging.Logger
+	base.Reconciler
 }
 
 //
 // Reconcile a Migration CR.
 // Note: Must not a pointer receiver to ensure that the
 // logger and other state is not shared.
 func (r Reconciler) Reconcile(request reconcile.Request) (result reconcile.Result, err error) {
-	fastReQ := reconcile.Result{RequeueAfter: FastReQ}
-	slowReQ := reconcile.Result{RequeueAfter: SlowReQ}
-	noReQ := reconcile.Result{}
-	result = noReQ
-
-	r.log = logging.WithName(
+	r.Log = logging.WithName(
 		names.SimpleNameGenerator.GenerateName(Name+"|"),
-		"migration",
+		"map",
```

Comment:
This looks like a possible cut and paste error

Reply:
Fixed.